third_party/poco/config.make

```diff
@@ -1,12 +1,12 @@
 # config.make generated by configure script
 POCO_CONFIG = Darwin64-clang-libc++
-POCO_BASE = /Users/nghiatran/Documents/Work/Toggl/toggldesktop/third_party/poco
-POCO_BUILD = /Users/nghiatran/Documents/Work/Toggl/toggldesktop/third_party/poco
+POCO_BASE = /Users/martinbriza/code/toggldesktop/third_party/poco
```

Comment:
Is it safe to be hard code? I assume that config.make is automatically generated from your current machine.

Reply:
Your machine was there before and I probably commited this by mistake. Let me remove any changes from `third_party` before merging.